static/styles/zulip.css

```diff
@@ -22,6 +22,9 @@ $sidebar_top: calc($header_height + $header_padding_bottom);
 $left_sidebar_width: 270px;
 $right_sidebar_width: 250px;
 
+/* Font sizes */
+$fifteen_pixels: 107.14%;
```

Comment:
We will have to use variables like these if we want to use percentages. We have decided on font size nomenclature yet, so just left it as fifteen_pixels.

Reply:
No we don’t; we can write `font-size: calc(15em / 14)`.

Reply:
Thanks, updated!